invokeai/frontend/web/src/features/modelManagerV2/subpanels/AddModelPanel/HuggingFaceFolder/HuggingFaceResults.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Button,
+  Divider,
+  Flex,
+  Heading,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+} from '@invoke-ai/ui-library';
+import { useAppDispatch } from 'app/store/storeHooks';
+import ScrollableContent from 'common/components/OverlayScrollbars/ScrollableContent';
+import { addToast } from 'features/system/store/systemSlice';
+import { makeToast } from 'features/system/util/makeToast';
+import type { ChangeEventHandler } from 'react';
+import { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PiXBold } from 'react-icons/pi';
+import { useInstallModelMutation } from 'services/api/endpoints/models';
+
+import { HuggingFaceResultItem } from './HuggingFaceResultItem';
+
+type HuggingFaceResultsProps = {
+  results: string[];
+};
+
+export const HuggingFaceResults = ({ results }: HuggingFaceResultsProps) => {
+  const { t } = useTranslation();
+  const [searchTerm, setSearchTerm] = useState('');
+  const dispatch = useAppDispatch();
+
+  const [installModel] = useInstallModelMutation();
+
+  const handleSearch: ChangeEventHandler<HTMLInputElement> = useCallback((e) => {
+    setSearchTerm(e.target.value.trim());
+  }, []);
+
+  const clearSearch = useCallback(() => {
+    setSearchTerm('');
+  }, []);
+
+  const handleAddAll = useCallback(() => {
+    for (const result of results) {
+      installModel({ source: result })
+        .unwrap()
+        .then((_) => {
+          dispatch(
+            addToast(
+              makeToast({
+                title: t('toast.modelAddedSimple'),
+                status: 'success',
+              })
+            )
+          );
+        })
+        .catch((error) => {
+          if (error) {
+            dispatch(
+              addToast(
+                makeToast({
+                  title: `${error.data.detail} `,
+                  status: 'error',
+                })
+              )
+            );
+          }
+        });
+    }
+  }, [results, installModel, dispatch, t]);
+
+  return (
+    <>
+      <Divider mt={4} />
+      <Flex flexDir="column" gap={2} mt={4} height="100%">
+        <Flex justifyContent="space-between" alignItems="center">
+          <Heading fontSize="md" as="h4">
+            {t('modelManager.availableModels')}
+          </Heading>
+          <Flex alignItems="center" gap="4">
+            <Button onClick={handleAddAll} isDisabled={results.length === 0}>
```

Comment:
I think this button (here and on folder search tab) would look better as `size="sm"`